app/views/addresses/index.html.erb

```diff
@@ -5,125 +5,99 @@
 <div class="text-center fs-5 fw-light my-4 ">
     <h5>Adress Details</h5>
 </div>
-<div class="row">
+<div class="row overflow-hidden">
 	<div class="col-md-8">
-		<!-- Button trigger modal -->
-		<button type="button" class="btn btn-primary" data-bs-toggle="modal" data-bs-target="#exampleModal">
-			Edit form
-		</button>
-		<!-- Modal -->
-<div class="modal fade" id="exampleModal" tabindex="-1" aria-labelledby="exampleModalLabel" aria-hidden="true">
-  <div class="modal-dialog">
-    <div class="modal-content">
-      <div class="modal-header">
-        <h5 class="modal-title" id="exampleModalLabel">Modal title</h5>
-        <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>
-      </div>
-      <div class="modal-body">
-			<form class="row g-3">
-			<div class="col-md-6">
-					<label for="inputEmail4" class="form-label">Email</label>
-					<input type="email" class="form-control" id="inputEmail4">
-			</div>
-			<div class="col-md-6">
-					<label for="inputPassword4" class="form-label">Password</label>
-					<input type="password" class="form-control" id="inputPassword4">
-			</div>
-			<div class="col-12">
-					<label for="inputAddress" class="form-label">Address</label>
-					<input type="text" class="form-control" id="inputAddress" placeholder="1234 Main St">
-			</div>
-			<div class="col-12">
-					<label for="inputAddress2" class="form-label">Address 2</label>
-					<input type="text" class="form-control" id="inputAddress2" placeholder="Apartment, studio, or floor">
-			</div>
-			<div class="col-md-6">
-					<label for="inputCity" class="form-label">City</label>
-					<input type="text" class="form-control" id="inputCity">
-			</div>
-			<div class="col-md-4">
-					<label for="inputState" class="form-label">State</label>
-					<select id="inputState" class="form-select">
-					<option selected>Choose...</option>
-					<option>...</option>
-					</select>
-			</div>
-			<div class="col-md-2">
-					<label for="inputZip" class="form-label">Zip</label>
-					<input type="text" class="form-control" id="inputZip">
-			</div>
-			<div class="col-12">
-					<div class="form-check">
-					<input class="form-check-input" type="checkbox" id="gridCheck">
-					<label class="form-check-label" for="gridCheck">
-							Check me out
-					</label>
+		<div class="container">
+			<div class="card border shadow-none">
+				<div class="row">
+					<div class="">
+						<% if current_user.addresses.present? %>
+							<div>
+
+							</div>
+							<!-- Button trigger modal -->
+							<button type="button" class="btn btn-primary" data-bs-toggle="modal" data-bs-target="#exampleModal">
+								Edit form1
+							</button>
+							<!-- Modal -->
+							<%= render partial:"modal" %>	
```

Comment:
It is right way to render a modal

Reply:
yes, this the right way to render the modal but if you the better way pls let me know